CHANGELOG.md

```diff
@@ -5,7 +5,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/global-121/121-platform/compare/v1.108.0...master)
+## [Unreleased](https://github.com/global-121/121-platform/compare/v1.108.3...master)
+
+## [1.108.3](https://github.com/global-121/121-platform/compare/v1.108.3...v1.108.2) - 2023-12-29
```

Comment:
I think all of these "compare"-links for v1.108 are in the wrong 'order'...

They should be: `https://github.com/global-121/121-platform/compare/<OLD_TAG>...<NEW_TAG>` (and not `NEW...OLD`)